src/components/audio-entry-details/_audio-entry-details.scss

```diff
@@ -55,8 +55,8 @@
 
     &.audio-entry-expanded {
       overflow: auto;
-      background: rgba(0, 0, 0, 0.7);
-      height: 100%;
+      background: linear-gradient(180deg, rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.7) 100%);
```

Comment:
Please attach screenshot in PR desc, anyway i don’t see any requirement for gradient

Reply:
this is the style when the size of the player is smaller. it should be like this so the part of the title will be more visible